script/src/known_bugs_checker.rs

```diff
@@ -0,0 +1,137 @@
+use crate::ScriptError;
+use byteorder::{ByteOrder, LittleEndian};
+use ckb_error::Error;
+use ckb_types::core::TransactionView;
+use ckb_vm::{
+    instructions::{extract_opcode, i, m, rvc, Instruction, Itype, Stype},
+    registers::RA,
+};
+use ckb_vm_definitions::instructions as insts;
+use goblin::elf::{section_header::SHF_EXECINSTR, Elf};
+
+const CKB_VM_ISSUE_92: &str = "https://github.com/nervosnetwork/ckb-vm/issues/92";
+
+pub struct KnownBugsChecker<'a> {
```

Comment:
`IllTransactionChecker` might be a better name.

Reply:
What does `Ill` mean?

Reply:
Means some transaction are considered "unhealthy".

Reply:
Oops, silly me😂😂😂

Reply:
IMO IllInstructionChecker is more clear.